x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/TransactionSelect.tsx

```diff
@@ -43,38 +46,39 @@ export const TransactionSelect: React.SFC<TransactionSelectProps> = ({
       <EuiSuperSelect
         valueOfSelected={selected}
         onChange={onChange}
-        options={types.map((type, i) => ({
-          value: type,
-          inputDisplay: type,
-          dropdownDisplay: (
-            <EuiFlexGroup justifyContent="spaceBetween">
-              <EuiFlexItem>
-                <EuiText>{type}</EuiText>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                {Boolean(
-                  existingJobs.find(
-                    job => job.jobId && job.jobId.includes(type)
```

Comment:
This check was prone to errors. Eg. if a service name contained the transaction type it would incorrectly match. Also problematic if a transaction .type was the contained in another, eg. "requests" and "web_requests"